charts/scalardl-audit/values.yaml

```diff
@@ -0,0 +1,239 @@
+    namespace: monitoring
+
+  grafanaDashboard:
+    # -- enable grafana dashboard
+    enabled: false
+    # -- which namespace grafana dashboard is located. by default monitoring
+    namespace: monitoring
+
+  # -- resources allowed to the pod
+  resources:
+    {}
+    # limits:
+    #   cpu: 100m
+    #   memory: 128Mi
+    # requests:
+    #   cpu: 100m
+    #   memory: 128Mi
+
+  # -- nodeSelector is form of node selection constraint
+  nodeSelector: {}
+
+  # -- Tolerations are applied to pods, and allow (but do not require) the pods to schedule onto nodes with matching taints.
+  tolerations: []
+
+  # -- the affinity/anti-affinity feature, greatly expands the types of constraints you can express
+  affinity: {}
+
+auditor:
+  # -- number of replicas to deploy
+  replicaCount: 3
+
+  scalarAuditorConfiguration:
+    # -- The contact points of the database such as hostnames or URLs
+    dbContactPoints: cassandra
+    # -- The port number of the contact points
+    dbContactPort: 9042
+    # -- The username of the database
+    dbUsername: cassandra
+    # -- The password of the database
+    dbPassword: cassandra
+    # -- The storage of the database: cassandra or cosmos
+    dbStorage: cassandra
+    # -- The port number of Auditor Server
+    auditorServerPort: 50051
+    # -- The port number of Auditor Privileged Server
+    auditorServerPrivilegedPort: 50052
+    # -- The port number of Auditor Admin Server
+    auditorServerAdminPort: 50053
+    # -- The log level of Scalar auditor
+    auditorLogLevel: INFO
+    # -- The holder ID of an Auditor certificate
+    auditorCertHolderId: auditor
+    # -- The version of an Auditor certificate
+    auditorCertVersion: 1
+
+  image:
+    # -- Docker image
+    repository: ghcr.io/scalar-labs/scalar-auditor
+    # -- Docker tag
+    version: 3.0.1
+    # -- Specify a imagePullPolicy
+    pullPolicy: IfNotPresent
+
+  # -- Optionally specify an array of imagePullSecrets. Secrets must be manually created in the namespace.
+  imagePullSecrets: [name: reg-docker-secrets]
+
+  strategy:
+    rollingUpdate:
+      # -- The number of pods that can be created above the desired amount of pods during an update
+      maxSurge: 25%
+      # -- The number of pods that can be unavailable during the update process
+      maxUnavailable: 25%
+    # -- New pods are added gradually, and old pods are terminated gradually, e.g: Recreate or RollingUpdate
+    type: RollingUpdate
+
+  # -- PodSecurityContext holds pod-level security attributes and common container settings
+  podSecurityContext:
+    {}
+    # fsGroup: 2000
+
+  # -- Setting security context at the pod applies those settings to all containers in the pod
+  securityContext:
+    {}
+    # capabilities:
+    #   drop:
+    #   - ALL
+    # readOnlyRootFilesystem: true
+    # runAsNonRoot: true
+    # runAsUser: 1000
+
+  service:
+    # -- service types in kubernetes
+    type: ClusterIP
+    annotations: {}
```

Comment:
Do we need to comment something like envoy service?

- https://github.com/scalar-labs/helm-charts/pull/21/files#diff-968e2d775678c536f4e5c8c7b81c8be561bf3b8acdcbd4d91c10ba00e4cd1b35R56-R57